scripts/globals/spells/bio_ii.lua

```diff
@@ -1,6 +1,6 @@
 -----------------------------------------
 -- Spell: Bio II
--- Deals dark damage that weakens an enemy's attacks and gradually reduces its HP.
+-- Deals dark damage that weakens an enemy's attacks and gradually reduces it's HP.
```

Comment:
its was correct. It's = it is. Reduces it is HP?
its = belong to it. Reduce the hp belonging to it? Sounds better.

Reply:
whatever the helptext on retail says /shrug

Reply:
Bio description text:
![image](https://user-images.githubusercontent.com/6871475/57195874-4a262e00-6f25-11e9-8f9d-0cbeff523289.png) (lowers an enemy is attacks..ok SE)

Dia description text:
![image](https://user-images.githubusercontent.com/6871475/57195887-60cc8500-6f25-11e9-911f-f593f1d2bbd6.png)